README.md

```diff
@@ -42,6 +43,10 @@ To successfully utilize this project first of all you need to clone or download
 - Twitter: [@EyasuTeshome](https://twitter.com/EyasuTeshome)
 - LinkedIn: [Eyasu Teshome](https://linkedin.com/in/EyasuTeshome)
 
+## Contributors
+👤 Faustin Makutano
+👤 Mehria-Rahmani
+
```

Comment:
- [ **OPTIONAL** ]**IMPORTANT**: Good job adding changes required by the previous reviewer. So, I noticed that you didn't add their social link address. Make sure you do so to make documentation more professional.